packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -43,6 +43,10 @@ const propTypes = {
    * A callback function to request focus from the containing component (e.g. modal).
    */
   requestFocus: PropTypes.func,
+  /**
+   * @private Internal prop for showing date picker.
+   */
+  shouldShowPicker: PropTypes.bool,
```

Comment:
This comment may need changing, noting it here.

Reply:
What changing would it need?

Reply:
Not sure! :) Usually my comments can be worded better, so I tagged them here for easier input.

Reply:
Ah gottchya! Maybe add for test / doc examples??